### ISSUE TITLE
Machine Learning Engineer

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#8123]
Machine Learning Engineer<br>
https://ift.tt/2hDyt8p<br>
<p><img src="https://we-work-remotely.imgix.net/logos/0006/6988/logo.gif?ixlib=rails-2.1.3&amp;w=190&amp;min-h=150" alt="Logo.gif?ixlib=rails 2.1"></p>
<p><strong>Headquarters:</strong> Singapore<br>
<strong>URL:</strong> <a href="https://pactly.ai">https://pactly.ai</a></p>
<div><span>Hi! We&rsquo;re Pactly, a Singapore-based LegalTech startup, using machine learning to automate the process of contract review and analysis. Pactly allows users to upload their contract and receive a report of the risky and unusual clauses in a matter of seconds.</span></div>
<div><br></div>
<div><span>For individuals and small teams signing contracts with BigCorp Inc., Pactly levels the playing field by offering an alternative type of check - especially if you are simply unable to pay the high cost of traditional legal services. On the other hand, our enterprise customers use Pactly as an efficient solution to review their high volume of routine contracts, reducing the repetitive and frustrating tasks that they are often obligated to perform manually.</span></div>
<div><br></div>
<div><span>At our core, we believe that all individuals and businesses spend way too much time and money reading contracts just to figure out whether we should sign them. We need a better, faster and more painless way to understand the contracts we sign and this has become our obsession.</span></div>
<div><br></div>
<div>
<span>We are looking for a machine learning engineer to join our small but scrappy team</span><span>. You will primarily be working on (a) improving our core machine learning algorithms which we use to identify key concepts in the contracts, (b) improving our data management and model training infrastructure, and (c) developing processes and tools to monitor and analyze model performance and data accuracy.</span>
</div>
<div><br></div>
<div><span>You should be a self-managing individual and unless you live in Singapore or would be keen to relocate, you would be working mainly remotely.</span></div>
<div><br></div>
<div>
<span>You should have:</span><br>
</div>
<ul>
<li>Strong ability in writing Python code for ML projects<br>
</li>
<li>Familiarity with deep learning frameworks (e.g. Tensorflow, Keras etc.)<br>
</li>
<li>Knowledge of a variety of machine learning techniques (decision tree classifiers, SVMs, artificial neural networks, etc.) and their real-world advantages/drawbacks<br>
</li>
<li>Experience with text classification projects and basic skills around common natural language processing tasks<br>
</li>
<li>Fluent in English with excellent written and verbal communication skills<br>
</li>
<li>A quick study, passionate about learning new technologies</li>
</ul>
<ul>
<li>Ability to work independently</li>
<li>Attitude: default to action</li>
</ul>
<div><br></div>
<div>
<span>Bonus if you have:</span><br>
</div>
<ul>
<li>Worked in a data science team in a growth stage startup<br>
</li>
<li>Know your way around AWS<br>
</li>
<li>Experience working with NoSQL databases, MongoDB, Elasticsearch<br>
</li>
<li>Previous remote working experience</li>
</ul>
<div><br></div>
<div>
<span>We offer:</span><br>
</div>
<ul>
<li>Full-time job, with flexible working hours<br>
</li>
<li>Long-term commitment with a fair equity package<br>
</li>
<li>Opportunity to work on high-impact ideas in an emerging space<br>
</li>
<li>Absolutely no corporate bureaucracy<br>
</li>
<li>A results-oriented team that&rsquo;s open, transparent and happy<br>
</li>
<li>Relocation benefits, should you wish to move to Singapore</li>
</ul>
<p><strong>To apply:</strong> If you feel you&rsquo;d be a good match, forward your LinkedIn profile along with a thoughtful cover letter to <a href="mailto:letsworktogether@pactly.ai">letsworktogether@pactly.ai</a>. Tell us at least about one relevant project and include a link to the project (if available). Note: We are looking for long-term, full-time employees to join the team so no applications from recruiters, agencies or contractors will be considered.</p>
<br>
<br>
via Remote Jobr Programming Jobs https://ift.tt/1WipnMk<br>
August 24, 2018 at 05:15AM
